requirements-minimal.txt

```diff
@@ -4,10 +4,11 @@ jsonschema>=3.2.0
 PyYAML>=5.4
 scipy>=1.4.1
 h5py>=3.9.0
-hdmf>=3.12.1
+hdmf>=3.12.2
```

Comment:
Required to avoid the deprecation warning on passing `**kwargs` instead of `data_io_kwargs`

Reply:
OK